languages/en.ini

```diff
@@ -1100,6 +1100,12 @@ Series = "Series"
 Set = "Set"
 shortlink_redirect = "You will be redirected in %%delay%% seconds..."
 show_filters_html = "Show filters (%%count%%)"
+show_less = "Show less"
+show_less_items = "Hide items"
+show_less_subjects = "Hide subjects"
```

Comment:
Also, any objection to calling these show_fewer_items and show_fewer_subjects to be less grammatically awkward?